charts/vm-import-controller/Chart.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v2
+name: vm-import-controller
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
nit: We can add a suffix `-dev` in master branch like
https://github.com/harvester/charts/blob/5449938d02bf8e32c45b0bf9ff4f76ba279fae81/charts/harvester-csi-driver/Chart.yaml#L21
And use a real version when porting the change to `release` branch.

Reply:
thanks @bk201. i have fixed this now.